### ISSUE TITLE
Remove <br> tags from index.js

### ISSUE THREAD

**Issue by will0684** [DTS-STN/Alpha-Site#88]
The [Web Accessibility Checklist](https://bati-itao.github.io/resources/a11ycheck-en.html) states we should be avoiding using <br> tags to create whitespace between elements.

The only page we have these tags on is index.js. 

Let's change it so that we are using CSS instead.
